.github/workflows/ci_servicex.yaml

```diff
@@ -85,12 +85,12 @@ jobs:
       - name: Extract tag name
         working-directory: ${{ matrix.app.dir_name }}
         shell: bash
-        run: echo "##[set-output name=imagetag;]$(echo sslhep/${{matrix.app.image_name}}:${GITHUB_REF##*/})"
+        run: echo "imagetag=$(echo sslhep/${{ matrix.app.image_name }}:${GITHUB_REF##*/})" >> $GITHUB_OUTPUT
```

Comment:
In the current existing

```yaml
      run: echo "##[set-output name=imagetag;]$(echo sslhep/${{matrix.app.image_name}}:${GITHUB_REF##*/})"
```

I'm not very clear on what the first `##` was intended to strip out, so this subset of changes might be missing the original point of things like https://github.com/ssl-hep/ServiceX/commit/9f269882ba4e4e22ab3aa239aa4d10973a26d2c1.

Reply:
I just found this PR. It should be encompassed in the one I made: 
https://github.com/ssl-hep/ServiceX/pull/603
Please feel free to review...